packages/next/src/build/noop-react-dom-server-legacy.ts

```diff
@@ -0,0 +1,11 @@
+export function renderToString() {
+  throw new Error(
+    `renderToString() is not supported when importing 'react-dom/server.edge' or 'react-dom/server.browser', use 'react-dom/server' instead.`
```

Comment:
suggesting `/server` will not work in edge environments I think. In edge environments the `/server` package will resolve to `/server.edge` and that will try to import the legacy browser build.